src/test/regress/expected/isolation_alter_role_propagation.out

```diff
@@ -0,0 +1,108 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1-begin s1-alter-role-1 s2-add-node s1-commit
+run_command_on_workers
+
+step s1-begin: 
+    SET citus.enable_alter_role_propagation to ON;
+    SELECT run_command_on_workers($$SET citus.enable_alter_role_propagation to ON;$$);
+    BEGIN;
+
+run_command_on_workers
+
+step s1-alter-role-1: 
+    ALTER ROLE alter_role_1 NOSUPERUSER;
+
+step s2-add-node: 
+    SELECT 1 FROM master_add_node('localhost', 57637);
+ <waiting ...>
+step s1-commit: 
+    COMMIT;
+
+step s2-add-node: <... completed>
+?column?       
+
+1              
+run_command_on_workers
+
+(localhost,57637,t,SET)
+
+starting permutation: s1-begin s1-add-node s2-alter-role-1 s1-commit
+run_command_on_workers
+
+(localhost,57637,t,"CREATE ROLE")
+step s1-begin: 
+    SET citus.enable_alter_role_propagation to ON;
+    SELECT run_command_on_workers($$SET citus.enable_alter_role_propagation to ON;$$);
+    BEGIN;
+
+run_command_on_workers
+
+(localhost,57637,t,SET)
+step s1-add-node: 
+    SELECT 1 FROM master_add_node('localhost', 57637);
```

Comment:
This node already exists in the cluster, so `master_add_node ` does nothing -- no-op. 

To make this test work,  we should remove the node on `setup`, and add it like this